examples/wiznet5k_simpleserver.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-FileCopyrightText: 2021 Adam Cummick
+#
+# SPDX-License-Identifier: MIT
+
+import board
+import busio
+import digitalio
+from adafruit_wiznet5k.adafruit_wiznet5k import WIZNET5K
+import adafruit_wiznet5k.adafruit_wiznet5k_socket as socket
+
+print("Wiznet5k SimpleServer Test")
+
+# For Adafruit Ethernet FeatherWing
+cs = digitalio.DigitalInOut(board.D10)
+# For Particle Ethernet FeatherWing
+# cs = digitalio.DigitalInOut(board.D5)
+spi_bus = busio.SPI(board.SCK, MOSI=board.MOSI, MISO=board.MISO)
+
+# Initialize ethernet interface
+eth = WIZNET5K(spi_bus, cs, is_dhcp=False)
```

Comment:
why is `is_dhcp` `False`?

Reply:
That's how I was testing it. In general, I figured for a webserver it would be preferable to have the IP be static.  It also shows that bind can be used to set the IP of the W5k interface.